sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -1,8 +1,23 @@
-import { SummaryEducationProgramDto } from "../../types";
+import { EducationProgramDto, SummaryEducationProgramDto } from "../../types";
 import HttpBaseClient from "./common/HttpBaseClient";
 
 export class EducationProgramApi extends HttpBaseClient {
-  public async createProgram(createProgramDto: any): Promise<void> {
+  public async getProgram(programId: number): Promise<void> {
+    try {
+      const response = await this.apiClient.get(
+        `institution/education-program/${programId}`,
+        this.addAuthHeader(),
+      );
+      return response.data;
```

Comment:
function is having a `void` promise and data is returning

Reply:
Fixed to any for now until we figure out the duplication issue with Sonar Cloud or we start to share the DTOs between web/api.